src/shared/Internal.Shared.Tasks/Internal.Shared.Tasks.proj

```diff
@@ -0,0 +1,43 @@
+﻿<Project>
```

Comment:
This looks like a regular class library project. Since Internal.AspNetCore.BuildTools.Tasks.csproj and Internal.AspNetCore.Sdk.csproj (through Internal.AspNetCore.Sdk.Tasks.csproj) both depend on this, what problem have you solved?